main.py

```diff
@@ -10,6 +10,7 @@
     "bot.cogs.utility.autopin",
     #"bot.cogs.utility.langwarn",
     "bot.cogs.utility.automod",
+    "bot.cogs.utility.rankup",
```

Comment:
what's this indentation?

Reply:
wdym?

Reply:
Ah just github being broken again sorry

Reply:
Lmao, I was gonna scream at py charm for fucking up my PEP 8